test/ex_doc/formatter/html/templates_test.exs

```diff
@@ -83,19 +110,23 @@ defmodule ExDoc.Formatter.HTML.TemplatesTest do
     assert content =~ ~r{.*\"id\":\s*\"CompiledWithDocs\".*\"functions\".*\"example/2\".*}ms
     assert content =~ ~r{.*\"id\":\s*\"CompiledWithDocs\".*\"macros\".*\"example_1/0\".*}ms
     assert content =~ ~r{.*\"id\":\s*\"CompiledWithDocs\".*\"functions\".*\"example_without_docs/0\".*}ms
-    assert content =~ ~r{.*"CompiledWithDocs.Nested\"}ms
+    assert content =~ ~r{.*"CompiledWithDocs.Nested\"}ms # fix emacs highlighting "
     refute content =~ ~r{.*\"exceptions\":}ms
     refute content =~ ~r{.*\"protocols\":}ms
   end
 
   test "listing page has README link if present" do
     content = Templates.sidebar_template(doc_config, [], [], [], true)
-    assert content =~ ~r{<a href="README.html">README</a>}
+    links = Floki.find(content, ".sidebar-mainNav li")
+
+    assert_text Enum.at(links, 0), "README"
   end
 
   test "listing page doesn't have README link if not present" do
     content = Templates.sidebar_template(doc_config, [], [], [], false)
-    refute content =~ ~r{<a href="README.html">README</a>}
+    links = Floki.find(content, ".sidebar-mainNav li")
+
+    assert_text Enum.at(links, 0), "Overview"
```

Comment:
What happens when someone change the order of the links in the menu? Imagine that first comes Overview and then README, this tests will pass but the README link will be present, right?